website/docs/cloud-docs/vcs/troubleshooting.mdx

```diff
@@ -87,6 +87,8 @@ The fix is to update the OAuth Callback URL in your VCS provider to use app.terr
 
 A workspace with no runs will not accept new runs from a VCS webhook. You must queue at least one run manually.
 
+A workspace will not process a webhook if the workspace previously processed a webhook with the same commit SHA and created a run. To trigger a run, create a new commit.
```

Comment:
We should mention that the user should look at the VCS Event page to determine if a duplicate commit was received and not processed.